lsp-bridge-call-hierarchy.el

```diff
@@ -0,0 +1,172 @@
+;;; lsp-bridge-call-hierarchy.el --- LSP bridge  -*- lexical-binding: t -*-
+
+(defvar lsp-bridge-call-hierarchy-name-width 40
+  "Width of the call name column in the call hierarchy buffer.")
+
+(defvar lsp-bridge-call-hierarchy-path-width 60
+  "Width of the path column in the call hierarchy buffer.")
+
+(defvar lsp-bridge-call-hierarchy-preview-height 30
+  "Height of the preview window. (set 0 to disable preview)")
+
+(defvar lsp-bridge-call-hierarchy--frame nil)
+
+(defvar lsp-bridge-call-hierarchy--index 0)
+
+(defvar lsp-bridge-call-hierarchy--overlay nil)
+
+(defvar lsp-bridge-call-hierarchy--popup-response nil)
+
+(defun lsp-bridge-incoming-call-hierarchy ()
+  (interactive)
+  (lsp-bridge-call-file-api "call_hierarchy"
+                            (lsp-bridge--position) t))
```

Comment:
After handler split with two class, we should change to `call_hierarchy_incoming_calls" here.